samples-generator/lib/createBuildingsTile.js

```diff
@@ -161,13 +195,20 @@ function generateBatchTableBinary(buildings) {
 
     var batchTableJson = {
         cartographic : {
+            name : 'CartographicBuffer',
             byteOffset : 0,
-            componentType : 'DOUBLE',
-            type : 'VEC3'
+            byteLength : cartographicBuffer.length,
+            componentType : 0x1406, // TODO: Logical error, not sure what to do, 'DOUBLE' was here originally but GLTF only supports Floats
```

Comment:
We'll have to use floats for now. I noticed this too in https://github.com/CesiumGS/glTF/pull/1